### PR TITLE
dev/core#1282 unit test for bug

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Unit test for https://github.com/civicrm/civicrm-core/pull/15541 written in review

Before
----------------------------------------
No test

After
----------------------------------------
Test

Technical Details
----------------------------------------
Last line I think *should* work but doesn't yet. Without the patch by @VangelisP I get a fatal error on the line before

Comments
----------------------------------------

### CODE REVIEW

File: tests/phpunit/api/v3/ProfileTest.php [hunk 1]
```diff
@@ -109,6 +112,29 @@ public function testProfileGet() {
...
+    $this->callAPISuccess('UFField', 'create', [
+      'uf_group_id' => $this->_profileID,
+      'field_name' => $this->getCustomFieldName('text'),
+      'visibility' => 'Public Pages and Listings',
+      'label' => 'My custom field',
+      'field_type' => 'Contact',
+    ]);
+    $this->callAPISuccess('Address', 'get', ['contact_id' => $this->_contactID, 'api.Address.create' => [$this->getCustomFieldName('text') => 'my field']]);
+    $result = $this->callAPISuccess('Profile', 'get', ['profile_id' => $this->_profileID, 'contact_id' => $this->_contactID])['values'];
+    // $this->assertEquals('my field', $result[$this->getCustomFieldName('text')]);
```
Comment (seamuslee001) [inline tests/phpunit/api/v3/ProfileTest.php]: @eileenmcnaughton are you able to re-enable this last line?

Comment (eileenmcnaughton) [reply tests/phpunit/api/v3/ProfileTest.php]: @seamuslee001 no - cos it still doesn't pass

Comment (eileenmcnaughton) [reply tests/phpunit/api/v3/ProfileTest.php]: but the line before failed without @VangelisP's patch merged & I can't figure out the rest

Comment (seamuslee001) [reply tests/phpunit/api/v3/ProfileTest.php]: Even tho you have merged the other PR?

Comment (eileenmcnaughton) [reply tests/phpunit/api/v3/ProfileTest.php]: yes - the line before that is the one the other PR fixed

**Pull request merged**